content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -155,10 +155,8 @@ used.
 
 If multiple metrics are specified in a HorizontalPodAutoscaler, this
 calculation is done for each metric, and then the largest of the desired
-replica counts is chosen. If any of these metrics cannot be converted
-into a desired replica count (e.g. due to an error fetching the metrics
-from the metrics APIs) and a scale down is suggested by the metrics which
-can be fetched, scaling is skipped. This means that the HPA is still capable
+replica counts is chosen. If all of these metrics cannot be converted
```

Comment:
I won't comment if that matches the actual algorithm here, but current phrasing creates confusion - "This means" looked like it matched "scale down" previously, but not anymore. If so - what is the meaning of the last sentence. Also - why example with API call error was removed?